contracts/GenericLender/GenericAave.sol

```diff
@@ -28,42 +32,75 @@ contract GenericAave is GenericLenderBase {
 
     IProtocolDataProvider public constant protocolDataProvider = IProtocolDataProvider(address(0x057835Ad21a177dbdd3090bB1CAE03EaCF78Fc6d));
     IAToken public aToken;
+    IStakedAave public constant stkAave = IStakedAave(0x4da27a545c0c5B758a6BA100e3a049001de870f5);
```

Comment:
Can we keep the minimum diff changes, there's seems to be several functions rearranged compared to the original
https://github.com/Grandthrax/yearnV2-generic-lender-strat/blob/master/contracts/GenericLender/GenericAave.sol

It would be faster to review if the diff was specific to only the changes

Reply:
ops sorry about that. didn't think about this when reordered the functions

commit 75180ea2e9edc678775d66eaca0c6f44ce518f85 have all the changes excluding function reordering